### ISSUE TITLE
Stormjewel gems are not unique-equipped

### ISSUE THREAD

**Issue by MadeinNoxus** [SallyWhitemane/Mograine#1150]
**Description:**
According to the 3.1 patch notes Stormjewels should be unique-equipped (https://wowwiki-archive.fandom.com/wiki/Patch_3.1.0_(undocumented_changes)) - ''Stormjewels, epic unique-equipped pre-cut gems were added. They can rarely be found within a [Bag of Fishing Treasures]". Apparently they are not unique-equipped and you can stack them.

**How to reproduce:**
Get for example 2 Solid Stormjewels and place them into crafted belt - you can see that they can be stacked.

**How it should work:**
You shouldn't be able to place more than 1 unique-equipped gem in your gear.

**Comment by MissZay** [comment]
stormjewels aren't unique-equipped in 3.3.5 (was changed in 3.2)

**Comment by Zeorgbugfinder** [comment]
> stormjewels aren't unique-equipped in 3.3.5 (was changed in 3.2)

YES because why they should be on Toc+ patch where all gems are same as stormjewel but they definetly should be unique equiped now on uld patch where are rare gems only
